test/template_sync_errors.go

```diff
@@ -4,9 +4,9 @@
 package test
 
 import (
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-	. "github.com/stolostron/governance-policy-framework/test/common"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
Like I said in a previous comment, dot imports of ginkgo and gomega should be allowed everywhere in the repo.